### PR TITLE
docs(examples/vpc): add a vpc peering basic example

### PR DESCRIPTION
**Pull request by ruwenqiang123**
**What this PR does / why we need it**:
add a vpc peering basic example
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
a vpc peering basic example
```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
None
```

### CODE REVIEW

File: examples/vpc/peering/main.tf [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  destination = "0.0.0.0/0"
+  type        = "peering"
+  nexthop     = huaweicloud_vpc_peering_connection.peering.id
+}
+
+resource "huaweicloud_vpc_peering_connection" "peering" {
+  name        = var.peer_conn_name
+  vpc_id      = huaweicloud_vpc.vpcA.id
+  peer_vpc_id = huaweicloud_vpc.vpcB.id
+}
```
Comment (Lance52259) [inline examples/vpc/peering/main.tf]: pls append a new line

File: examples/vpc/peering/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Basic VPC peering
+
+This example provisions a  VPC peering connection is used to connect subnet A in VPC A to subnet B in VPC B,
+the local VPC is VPC A, and the peer VPC is VPC B.
```
Comment (Lance52259) [inline examples/vpc/peering/README.md]:
```suggestion
This example provisions a VPC peering connection is used to connect subnet A in VPC A to subnet B in VPC B,
the local VPC is VPC A, and the peer VPC is VPC B.
```

File: examples/vpc/peering/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+* a VPC instance with a subnet:
+  > Create a VPC A instance with subnet A and create a VPC B instance with subnet B.
+
+* a VPC peering:
+  > The local VPC is VPC A, and the peer VPC is VPC B.
+  > Add a route in the route table of VPC B. Set Destination to 0.0.0.0/0 and Next Hop to the created VPC peering
```
Comment (Lance52259) [inline examples/vpc/peering/README.md]:
```suggestion
  > Add a route in the route table of VPC B. Set destination to 0.0.0.0/0 and next hop to the created VPC peering
```

File: examples/vpc/peering/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+```shell
+terraform init
+terraform plan
+terraform apply
+terraform destroy
+```
+
+The creation of the VPC and VPC peering take about a few moment. After the creation is successful, the provisioner
+starts to run.
```
Comment (Lance52259) [inline examples/vpc/peering/README.md]: there is no provisioner in your script

Comment (ruwenqiang123) [reply examples/vpc/peering/README.md]: done

File: examples/vpc/peering/README.md (hunk 3 above)
Comment (ruwenqiang123) [reply examples/vpc/peering/README.md]: done

File: examples/vpc/peering/README.md (hunk 2 above)
Comment (ruwenqiang123) [reply examples/vpc/peering/README.md]: done

File: examples/vpc/peering/main.tf (hunk 1 above)
Comment (ruwenqiang123) [reply examples/vpc/peering/main.tf]: done

**Pull request merged**